OrbitGl/PresetsDataView.cpp

```diff
@@ -18,15 +18,57 @@
 #include "absl/strings/str_format.h"
 
 using orbit_client_protos::PresetFile;
+using orbit_client_protos::PresetInfo;
 
 PresetsDataView::PresetsDataView() : DataView(DataViewType::kPresets) {}
 
+std::string PresetsDataView::GetModulesList(const std::vector<ModuleView>& modules) const {
+  std::string modules_list;
+  for (const auto& module : modules) {
+    if (modules_list.empty()) {
```

Comment:
I think you can make this (and below) a bit more elegant by using absl::StrJoin (https://abseil.io/tips/36):
```
std::vector<std::string_view> modules_strings;
for (const auto& module : modules) {
  modules_strings.push_back(module.module_name);
}
return absl::StrJoin(modules_strings.begin(), modules_strings.end(), "\n");

```
}

Reply:
Done, thanks